src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -208,8 +220,88 @@ public async Task GetInfoPackage()
             }
         }
 
-        private async Task<Stream> GetDebugInfoFromNodeAsync(JsonOperationContext jsonOperationContext,
-            string url, IEnumerable<string> databaseNames, X509Certificate2 certificate)
+        private void DumpStacktraces(ZipArchive archive, string prefix)
+        {
+            var stacktraces = archive.CreateEntry($"{prefix}/stacktraces.json", CompressionLevel.Optimal);
+
+            var tempPath = ServerStore.Configuration.Storage.TempPath?.FullPath ?? Path.GetTempPath();
+
+            var ravenDebugExec = Path.Combine(AppContext.BaseDirectory, "Raven.Debug.exe");
+            var ravenDebugOutput = Path.Combine(tempPath, "stacktraces.json");
```

Comment:
Any reason we aren't just pumping this directly to the console as the output of `Raven.Debug`?

Reply:
Was thinking that there was, because it was done like that in 3.5, but... it works fine. Writing the output to the zip entry now.